### ISSUE TITLE
ReferenceError: DBTableEditor is not defined

### ISSUE THREAD

**Issue by akela92** [AccelerationNet/wp-db-table-editor#26]
Hello everyone:

I've been using this plugin for a few months now and I couldn't be happier, it is a great plugin. But for a few days nothing would load. I can't see the excel table or the icons. The browser outputs this error: 

> ReferenceError: DBTableEditor is not defined

I've tried different solutions but none worked.
The weird thing is that I didn't change anything, it just stopped working.

Thanks in advance

**Comment by bobbysmith007** [comment]
Hrm... Sorry to hear that it has stopped working for you.  Perhaps an upgrade had a bug in it?  What version of the plugin are you running?  Have you upgraded since it was running?  What were the different solutions you have tried?  (I currently have a working DBTE on my test site - so I think the basics are covered).

On first blush it sounds to me like some javascript error is preventing the whole thing from loading.  Which means that it could be unrelated to anything this plugin is doing (any javascript error could cause this).  The most direct debugging would be to open up the developer console (F12) and watch what errors are occurring.  

Hopefully we can get you going again soon.

**Comment by akela92** [comment]
Hello @bobbysmith007,
I am currently using version 1.5.3. I didn't upgrade the plugin since I installed it a few months ago.
It is indeed a javascript error as you said. I tried to rollback to a previous version of our plugin (we are using part of the functionality of your plugin inside one of the plugins we are developing) and I still get the same error.
From what I see in the console it looks like the error has nothing to do with your plugin, but with other things we have installed. The core error seems to be that for some reason our `load-scripts.php` is not running correctly.
Thanks for the fast answer and great plugin.
